src/main/java/frc/robot/subsystems/Loading.java

```diff
@@ -0,0 +1,79 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.LoadingConstants;
+
+public class Loading extends SubsystemBase {
+  /**
+   * Creates a new Loading.
+   */
+
+  private final CANSparkMax m_loadingMotor;
+  private final CANSparkMax m_cellevatorMotor1;
+  private final CANSparkMax m_cellevatorMotor2;
```

Comment:
These should be more descriptive variables, as the two motors do very different things.  Try using the names from the Bross Parts Tracker, Loader and Holder.

Reply:
we changed the names to loader and holder to be more specific